recipes-devtools/nsight-systems/nsight-systems-cli_2020.5.3.17-1.bb

```diff
@@ -0,0 +1,53 @@
+SUMMARY = "NVIDIA Nsight Systems is a multi-core CPU sampling profiler that \
```

Comment:
This is long enough that it should probably be `DESCRIPTION` rather than `SUMMARY`.

Reply:
OK
Agree, I will change it to `DESCRIPTION` instead